site/src/pages/CreateUserPage/CreateUserForm.tsx

```diff
@@ -124,6 +129,17 @@ export const CreateUserForm: FC<
             fullWidth
             label={Language.usernameLabel}
           />
+          <TextField
+            {...getFieldHelpers("name")}
+            onBlur={(e) => {
+              e.target.value = e.target.value.trim();
+              form.handleChange(e);
+            }}
+            autoComplete="name"
+            fullWidth
+            label={Language.nameLabel}
+            helperText={Language.nameHelperText}
```

Comment:
I think "Full name" is a very explicit label so we don't need a helper text. We usually use helper text in the UI to help users pass the correct values. Common use cases:
- Fields with intrinsic validation. Eg. `The password should contain one letter, one number,...`
- Fields with specific formatting. Eg. `mm/dd/YYYY`

Reply:
removed